include/deal.II/base/data_out_base.h

```diff
@@ -2177,138 +2347,34 @@ class DataOutInterface
                           const std::vector<std::string> &piece_names) const;
```

Comment:
should this be deprecated and the documentation be removed (just referencing the static version)?